utils/miller/wrapper.py

```
@@ -0,0 +1,81 @@
+    "nidx",
+    "pprint",
+    "usv",
+    "usvlite",
+    "xtab",
+    "dkvp",
+]
+
+
+io_opts = ""
+### INPUT
+inputs = Path(snakemake.input[0])
```

Ruff (0.8.2)

Undefined name `snakemake`

(F821)